app/assets/javascripts/discourse/tests/integration/components/plugin-outlet-test.js

```diff
@@ -103,6 +113,71 @@ module("Integration | Component | plugin-outlet", function (hooks) {
     );
   });
 
+  test("Can act as a wrapper around core implementation", async function (assert) {
+    const consoleErrorStub = sinon.stub(console, "error");
+
+    this.set("shouldDisplay", false);
+    this.set("yieldCore", false);
+    this.set("enableClashingConnector", false);
+
+    extraConnectorClass("outlet-with-default/my-connector", {
+      shouldRender(args) {
+        return args.shouldDisplay;
+      },
+    });
+
+    extraConnectorClass("outlet-with-default/clashing-connector", {
+      shouldRender(args) {
+        return args.enableClashingConnector;
+      },
+    });
+
+    await render(
+      hbs`
+        <PluginOutlet @name="outlet-with-default" @outletArgs={{hash shouldDisplay=this.shouldDisplay yieldCore=this.yieldCore enableClashingConnector=this.enableClashingConnector}}>
+          <span class='result'>Core implementation</span>
+        </PluginOutlet>
```

Comment:
(off topic 😛) I wonder if we can make `prettier-plugin-ember-template-tag` work with `hbs`-tagged strings

Reply:
pretty neat indeed